extensions/ots/message_configuration/templates/messaging_help.html

```diff
@@ -0,0 +1,53 @@
+{% load i18n %}
+
+<p>
+  {% trans "Here you can configure message templates for different messages hypha sends out.  You can configure the footer and header (which will apepar on each email), as we well as configure whether hypha sends out messages by default." %}
+</p>
+<p>
+  {% trans "Click 'Add' at the bottom to configure a template for an outgoing message type." %}
+</p>
+<p>
+  {% trans "Variables that will get replaced follow.  Use the uppercase name (before the colon) in any subject, email message, or slack message." %}
+</p>
+
+<p>{% trans "When the submission is available, the following are available:" %}</p>
+
+<ul>
+  <li>{% trans "SUBMISSION_URL: The URL in the hypha system" %}</li>
+  <li>{% trans "SUBMISSION_FUND_TITLE:  The title of the fund" %}</li>
+  <li>{% trans "SUBMISSION_TITLE: The title of the submission" %}</li>
+  <li>{% trans "SUBMISSION_USER_NAME: The user name of the submitter" %}</li>
+  <li>{% trans "SUBMISSION_USER_EMAIL: The email of the submitter" %}</li>
+  <li>{% trans "SUBMISSION_CONFIRMATION_TEXT_EXTRA: Extra text generated by hypha" %}</li>
+</ul>
+<p>{% trans "When the determination is the subject of the email, the following are also available:" %}</p>
+<ul>
+  <li>{% trans "DETERMINATION_URL: The URL of the determination" %}</li>
+  <li>{% trans "DETERMINATION_MESSAGE: The determination message as configured for the fund" %}</li>
+  <li>{% trans "DETERMINATION_OUTCOME: The outcome of the determination" %}</li>
+</ul>
+<p>{% trans "When a project is the subject of the email, the following are also available:" %}</p>
+<ul>
+  <li>{% trans "PROJECT_TITLE: The title of the project" %}</li>
+  <li>{% trans "PROJECT_URL: The URL of the project" %}</li>
+  <li>{% trans "PROJECT_LEAD_EMAIL: The project lead email" %}</li>
+  <li>{% trans "PROJECT_LEAD_NAME: The project lead name" %}</li>
+</ul>
+<p>{% trans "When an invoice is the subject of the email, the following are also available:" %}</p>
+<ul>
+  <li>{% trans "INVOICE_URL: " %}</li>
+</ul>
+<p>{% trans "The following system variables are always available:" %}</p>
+<ul>
+  <li>{% trans "ORG_EMAIL: " %}</li>
+  <li>{% trans "ORG_SHORT_NAME: " %}</li>
+  <li>{% trans "ORG_LONG_NAME: " %}</li>
+  <li>{% trans "ORG_URL: " %}</li>
+  <li>{% trans "ORG_GUIDE_URL: " %}</li>
+</ul>
+
+{% for id, default in self.email_messages.items %}
+  <div class="is-hidden" attr-email-id="{{ id }}">
```

Comment:
Can we just use "hidden" attribute instead of adding a class and a custom css file?

Reply:
Sure!